File: lab-2/src/pokemons/Gible.java
package pokemons;

import moves.*;
import ru.ifmo.se.pokemon.*;

public class Gible extends Pokemon {
    public Gible (String name, int level) {
       super(name, level);
       setStats(58,70,45,40,45,42);
       setType(Type.DRAGON, Type.GROUND);
       setMove(new Facade(), new Swagger());
    }
}

File: lab-2-kotlin/src/pokemons/Gible.kt
package pokemons

import moves.*

import ru.ifmo.se.pokemon.*

open class Gible(name: String?, level: Int) : Pokemon(name, level) {
    init {
        setStats(58.0, 70.0, 45.0, 40.0, 45.0, 42.0)
        setType(Type.DRAGON, Type.GROUND)
        setMove(Facade(), Swagger())
    }
}